tests/client/conftest.py

```diff
@@ -69,7 +69,7 @@ def is_using_real_services(request):
 
 @pytest.fixture(scope='session')
 def client():
-    return ScrapinghubClient(auth=TEST_ADMIN_AUTH,
+    return ScrapinghubClient(apikey=TEST_ADMIN_AUTH,
```

Comment:
@vshlapakov  just noticed you replaced `auth` with `apikey`. note that in some cases we pass colon separated auth string to the HubstorageClient, do you think this will never happen here? 

https://github.com/scrapinghub/scrapinghub-entrypoint-scrapy/blob/9e5c4f79f4fa2442cedf7e13185c7087961f6b46/sh_scrapy/env.py#L53-L53